.github/workflows/docker-build.yml

```diff
@@ -1,60 +1,81 @@
+# Build and push Image
 name: Build and push Image
+
 on:
   push:
-    branches:
-      - main
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
+      - name: Set tag
+        id: set-tag
+        run: |
+          SHORT_SHA=$(git rev-parse --short HEAD)
+          if [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
+            echo "::set-output name=tag1::latest"
+            echo "::set-output name=tag2::commit-$SHORT_SHA"
+          else
+            echo "::set-output name=tag1::dev"
+            echo "::set-output name=tag2::commit-$SHORT_SHA"
```

Comment:
```
            echo "::set-output name=tag2::commit-$SHORT_SHA"
```

can be executed once, when we're pushing to the non-main branch. The latest tag is only needed when we do a tagged release actually. We should offer a `latest` tag, just because we've pushed something to main. We shouldn't even build an image when we merge to main, to be honest. The main branch should just depict the current state of the developement. It needn't offer a stable artifact (a docker image and a helm chart); that's what tagged commits are for.

Reply:
Yeah, makes totally sense. Created #19